src/qservkafka/storage/votable.py

```diff
@@ -172,7 +173,12 @@ def _encode_char_column(
         bytes
             Serialized representation of the column.
         """
-        value_str = "" if value_raw is None else str(value_raw)
+        if isinstance(value_raw, datetime):
+            value_str = value_raw.strftime("%Y-%m-%dT%H:%M:%S")
```

Comment:
I'm assuming our timestamps are all UTC? Otherwise I'd suggest we normalize to UTC but I imagine  we shouldn't have to right?

Reply:
You have to go out of your way to store a time zone in a MySQL database, and I don't think we're using the data type that would do that. That is indeed an assumption that we're making. I'll mention that on Slack.